backend/src/auth/session_id.rs

```diff
@@ -86,14 +87,19 @@ impl SessionId {
             // anything, so something like "link to `/realm/delete`" is not a
             // thing for Tobira.
             .same_site(cookie::SameSite::Lax)
-            .finish()
+            .finish();
+
+        // Expire the cookie at the appropriate time
+        cookie.set_max_age(max_age);
+
+        cookie
```

Comment:
Why not use [`CookieBuilder::max_age`](https://docs.rs/cookie/0.15.1/cookie/struct.CookieBuilder.html#method.max_age)?

Reply:
Because that takes a `Duration`, not an `Option<Duration>`, which was originally the type of that argument. Will change.